core/src/main/java/org/apache/iceberg/util/ParallelIterable.java

```diff
@@ -34,6 +35,8 @@
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 
 public class ParallelIterable<T> extends CloseableGroup implements CloseableIterable<T> {
+  public static final int ITERATOR_QUEUE_SIZE = SystemConfigs.ITERATOR_QUEUE_SIZE.value();
```

Comment:
any particular reason to make this public? This doesn't seem to be used anywhere outside of this class. Also you probably could just use `SystemConfigs.ITERATOR_QUEUE_SIZE.value()` directly in L59

Reply:
It's a mistake.